buildSrc/src/main/java/com/uber/okbuck/core/util/ProjectUtil.java

```diff
@@ -224,4 +224,19 @@ public static boolean canHaveSources(File dependencyFile) {
         && ImmutableList.of(JAR, AAR)
             .contains(FilenameUtils.getExtension(dependencyFile.getName()));
   }
+
+  public static boolean getPropertyAsBoolean(Project project, String propertyName){
+    if(project.hasProperty(propertyName)){
+      return Boolean.parseBoolean((String)project.getProperties().get(propertyName));
+    }
+    return false;
+  }
+
+  @Nullable
+  public static String getPropertyAsString(Project project, String propertyName, String defaultValue){
+    if(project.hasProperty(propertyName)){
+      return (String)project.getProperties().get(propertyName);
+    }
+    return defaultValue;
+  }
```

Comment:
we should use OkBuckExtension to define these properties example: https://github.com/uber/okbuck/blob/master/buildSrc/src/main/java/com/uber/okbuck/extension/OkBuckExtension.java#L27

and have it false by default.